bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -182,8 +182,8 @@ defaultMeetingLayout=CUSTOM_LAYOUT
 #
 # native2ascii -encoding UTF8 bigbluebutton.properties bigbluebutton.properties
 #
-defaultWelcomeMessage=Welcome to <b>%%CONFNAME%%</b>!<br><br>For help on using BigBlueButton see these (short) <a href="https://bigbluebutton.org/teachers/tutorials/"><u>tutorial videos</u></a>.<br><br>To join the audio bridge click the speaker button.  Use a headset to avoid causing background noise for others.
-defaultWelcomeMessageFooter=This server is running <a href="https://bigbluebutton.org/" target="_blank"><u>BigBlueButton</u></a>.
+defaultWelcomeMessage=Welcome to %%CONFNAME%%! \n For help on using BigBlueButton see these (short) [Tutorial videos](https://bigbluebutton.org/teachers/tutorials/). \n To join the audio bridge click the speaker button.  Use a headset to avoid causing background noise for others.
```

Comment:
```suggestion
defaultWelcomeMessage=Welcome to **%%CONFNAME%%**!\n\nFor help on using BigBlueButton see these (short) [Tutorial videos](https://bigbluebutton.org/teachers/tutorials/).\n\nTo join the audio bridge click the speaker button.  Use a headset to avoid causing background noise for others.
```